integration_tests/test_list_03.py

```diff
@@ -24,6 +24,11 @@ def test_list_insert_02(x: list[i32], n: i32) -> list[i32]:
 
     return x
 
+def test_list_insert_03():
+    l1:list[str] = ["a","b","c","d"]
+    l1.insert(6,"e")
+    assert l1 == ['a', 'b', 'c', 'd', 'e']
+
```

Comment:
```suggestion
    l1:list[str] = ["a", "b", "c", "d"]
    l1.insert(6,"e")
    assert l1 == ['a',  'b',  'c',  'd',  'e']
    l1.insert(18, "f")
    assert l1 == ['a', 'b', 'c', 'd', 'e', 'f']
```

Reply:
Okay I'll add this quick. 
Also the logs fail, does asserting like this work or should I try asserting each element in a for loop ?

Reply:
Asserting each element should be better option for making sure our list algorithms are working as expected. But as a feature direct comparison of lists should also work (but that's a separate issue).

Reply:
As of now, all the integration tests do element wise comparison (possibly because direct assertion is an issue). I'll use a for loop now.

Reply:
I have updated the test accordingly.